JudoPayDotNet/Http/HttpClientWrapper.cs

```diff
@@ -14,6 +15,8 @@ namespace JudoPayDotNet.Http
     // ReSharper disable UnusedMember.Global
     internal class HttpClientWrapper : IHttpClient
     {
+        internal static readonly string SdkUserAgent = "DotNetSDK/" + new AssemblyName(typeof(HttpClientWrapper).GetTypeInfo().Assembly.FullName).Version;
```

Comment:
good point - not sure if it's available in PCL though, certainly the machine O/S isn't